clevertap-core/src/test/java/com/clevertap/android/sdk/CleverTapAPITest.kt

```diff
@@ -21,381 +22,299 @@ class CleverTapAPITest : BaseTestCase() {
-                        .thenReturn(corestate)
-                CleverTapAPI.getDefaultInstance(application)
-                CleverTapAPI.changeCredentials("acct123", "token123", "eu")
-                val instance = ManifestInfo.getInstance(application)
-                assertNotEquals("acct123", instance.accountId)
-                assertNotEquals("token123", instance.acountToken)
-                assertNotEquals("eu", instance.accountRegion)
+        setupBasicTestWithAny {
+            val expectedAccountId = "acct123"
+            val expectedToken = "token123"
+            val expectedRegion = "eu"
+            CleverTapAPI.getDefaultInstance(application)
+
+            val manifestInfo = ManifestInfo.getInstance(application)
+            // Act
+            CleverTapAPI.changeCredentials(expectedAccountId, expectedToken, expectedRegion)
+
+            // Assert
+            with(manifestInfo) {
+                assertNotEquals(expectedAccountId, accountId)
+                assertNotEquals(expectedToken, acountToken)
+                assertNotEquals(expectedRegion, accountRegion)
             }
         }
     }
 
     @Test
     fun test_changeCredentials_whenDefaultConfigNull_credentialsMustChange() {
-        mockStatic(CTExecutorFactory::class.java).use {
-            `when`(CTExecutorFactory.executors(any())).thenReturn(
-                    MockCTExecutors(
-                            cleverTapInstanceConfig
-                    )
-            )
+        setupMockExecutors {
+            // Arrange
+            val expectedAccountId = "acct123"
+            val expectedToken = "token123"
+            val expectedRegion = "eu"
             CleverTapAPI.defaultConfig = null
-            CleverTapAPI.changeCredentials("acct123", "token123", "eu")
-            val instance = ManifestInfo.getInstance(application)
-            assertEquals("acct123", instance.accountId)
-            assertEquals("token123", instance.acountToken)
-            assertEquals("eu", instance.accountRegion)
+
+            // Act
+            CleverTapAPI.changeCredentials(expectedAccountId, expectedToken, expectedRegion)
+
+            // Assert
+            val manifestInfo = ManifestInfo.getInstance(application)
+            with(manifestInfo) {
+                assertEquals(expectedAccountId, accountId)
+                assertEquals(expectedToken, acountToken)
+                assertEquals(expectedRegion, accountRegion)
+            }
         }
     }
 
     @Test
     fun test_createNotification_whenInstancesNull__createNotificationMustBeCalled() {
-
-        mockStatic(CTExecutorFactory::class.java).use {
-            `when`(CTExecutorFactory.executors(any())).thenReturn(
-                    MockCTExecutors(
-                            cleverTapInstanceConfig
-                    )
+        setupBasicTestWithAny {
+            val bundle = Bundle()
+            val lock = Object()
+            //CleverTapAPI.getDefaultInstance(application)
```

Comment:
Commented code

Reply:
fixed in https://github.com/CleverTap/clevertap-android-sdk/commit/5a2f2c022aac769fc693f5ff6400bc6bfde61628